examples/tornjak/README.md

```diff
@@ -40,3 +40,39 @@ kubectl -n spire-server port-forward service/spire-tornjak-frontend 3000:3000
 You can now access Tornjak at [localhost:3000](http://localhost:3000).
 
 See [values.yaml](./values.yaml) for more details on the chart configurations to achieve this setup.
+
+## Tornjak and Ingress
+
+Obtain the Ingress subdomain (this process is cloud provider specific) and assign it
+to the `APP_SUBDOMAIN` env. variable, then run the deployment with your values files:
+
+```shell
+export APP_SUBDOMAIN=$(kubectl get dns cluster -o jsonpath='{ .spec.baseDomain }')
+```
```

Comment:
This I think only works on certain clusters. It doesn't work for crc / minikube I think.

Also, it should probably print it out?